var/spack/repos/builtin/packages/fontconfig/package.py

```diff
@@ -38,9 +38,9 @@ class Fontconfig(AutotoolsPackage):
     depends_on('font-util', type='build')
 
     def configure_args(self):
-        args = ["--prefix=%s" % prefix,
+        font_path = join_path(self.spec['font-util'].prefix, 'share', 'fonts')
+
+        return ["--prefix={0}".format(self.prefix),
                 "--enable-libxml2",
                 "--disable-docs",
-                "--with-default-fonts=%s" %
-                spec['font-util'].prefix + "/share/fonts"]
```

Comment:
For reference, this is the line at which the undefined variable error was occurring (`spec` doesn't exist in this scope).